src/main/java/utils/RandomGenerator.java

```diff
@@ -15,4 +17,9 @@ public static String randomName() {
         return name.replaceAll("-", "");
     }
 
+    public static String randomDate(){
+        String date = new SimpleDateFormat("MM/dd/yyyy").format(new Date());
+        return date;
+    }
+
```

Comment:
This method is always going to return today's date, it's not random. Also, you only use it once in your code and it can be replaced with a constant